### ISSUE TITLE
add support for passage

### ISSUE THREAD

**Issue by ayushnix** [ayushnix/tessen#8]
Filippo Valsorda has released a fork of pass called [passage](https://github.com/FiloSottile/passage) which uses age instead of gpg.

**Comment by ayushnix** [comment]
I'm not sure if I want to add support for passage until [this](https://github.com/FiloSottile/age/discussions/244) issue is resolved. [This](https://github.com/str4d/age-plugin-yubikey) plugin adds support for yubikeys but even if you use this, you'd have to decrypt your passwords each and every time. Not to mention that there's no support for ad-hoc decryption when needed like pinentry menu does.

**Comment by fbergroth** [comment]
I did a test, and tessen seemed to worked fine with passage+age-plugin-yubikey by doing `sed -i s/gpg/age tessen` and having `PASSWORD_STORE_DIR=~/.passage/store`.

I wanted to avoid having to mess with gpg, but you have to re-enter the pin each time without an agent as you pointed out, which makes it a bit inconvenient. I'll bite the bullet and setup gpg keys instead.

**Comment by ayushnix** [comment]
Yeah, the lack of an agent and a pinentry like program for ad-hoc decryption makes passage support somewhat pointless for now. If this changes in the future, I'd be happy to work on it.
